apps/address-book/app/components/App/Entities.js

```diff
@@ -33,31 +33,30 @@ const Entities = ({ entities, onNewEntity, onRemoveEntity }) => {
   } else {
     return (
       <DataView
-        mode="table"
-        fields={[ 'Entity', '' ]}
+        mode="adaptive"
+        fields={[ 'Name', 'Address', 'Type' ]}
```

Comment:
This PR is now out of date. These changes were part of `addressbook-design-check`, which has now been merged to `dev` (#1214). Please rebase this branch with `dev` so that it's more clear what changed as part of the contract upgrade.

Reply:
It seems the code you are pointing has something wrong with the DataView and it broke my previous tested working code:

![image](https://user-images.githubusercontent.com/5030059/64636255-9f0d1000-d401-11e9-95c7-d38128b952d3.png)

I rebased as you suggested, but now I need to put some extra time debugging and fixing the DataView implementation to be able to have a working version of the app

Reply:
Fixed! It was easier than I thought, we just needed to rename some of the fields to adapt to the new `entry` object format